File: Python developer/step03/3.02/3.2.05.py
'''

Вам требуется подобрать коробку, для вашего товара. Известно, что ширина, высота и глубина коробки – целые числа.

Вашей программе даются на вход 3 числа с плавающей точкой – ширина, высота и глубина товара. Вам требуется подобрать для этого товара коробку. 
Выведите ширину, высоту и глубину коробки.

Sample Input 1:

8.4
6.3
2.9

Sample Output 1:

9
7
3

Sample Input 2:

1.0
2.0
3.0

Sample Output 2:

1
2
3

Sample Input 3:

1.01
2.003
4.006

Sample Output 3:

2
3
5

'''

import math

a, b, c = float(input()), float(input()), float(input())
print(math.ceil(a))
print(math.ceil(b))
print(math.ceil(c))

File: Diving into JavaScript: For Beginners/step03/3.02/3.2.05.js
/*

Напишите программу для оценки настроения на основе введенного пользователем числа от 1 до 10.

    1-3 - программа выводит "Плохое" 
    4-7 - программа выводит "Нормальное"
    8-10 - программа выводит "Хорошее"

Если пользователь ввел число вне диапазона, программа выводит "Число вне диапазона"

Sample Input 1:

10

Sample Output 1:

Хорошее

Sample Input 2:

3

Sample Output 2:

Плохое

Sample Input 3:

5

Sample Output 3:

Нормальное

Sample Input 4:

11

Sample Output 4:

Число вне диапазона

*/

let mood = Number(prompt());

if (mood >= 1 && mood <= 3) {
    console.log('Плохое');
}
else if (mood >= 4 && mood <= 7) {
    console.log('Нормальное');
}
else if (mood >= 8 && mood <= 10){
    console.log('Хорошее');
}
else {
    console.log('Число вне диапазона');
}
